graphql/src/Mu/GraphQL/Annotations.hs

```diff
@@ -110,11 +108,14 @@ instance (ReflectValueConst x, ReflectValueConstList xs)
     = reflectValueConst (Proxy @x) : reflectValueConstList (Proxy @xs)
 
 class ReflectValueConstObject xs where
-  reflectValueConstObject :: proxy xs -> [GQL.ObjectFieldG GQL.ValueConst]
+  reflectValueConstObject :: proxy xs -> [GQL.ObjectField GQL.ConstValue]
 instance ReflectValueConstObject '[] where
   reflectValueConstObject _ = []
 instance (KnownSymbol a, ReflectValueConst x, ReflectValueConstObject xs)
          => ReflectValueConstObject ( '(a, x) ': xs) where
   reflectValueConstObject _
-    = GQL.ObjectFieldG (coerce $ T.pack $ symbolVal (Proxy @a)) (reflectValueConst (Proxy @x))
+    = GQL.ObjectField (T.pack $ symbolVal (Proxy @a))
+                      (GQL.Node (reflectValueConst (Proxy @x)) zl)
+                      zl
       : reflectValueConstObject (Proxy @xs)
+    where zl = GQL.Location 0 0
```

Comment:
I don't understand this bit 🤔

Reply:
This library requires a location for each node, so I'm creating a fake one. I'm planning to take advantage of these locations in error messages in the future, but now I just wanted to get this out.